app/ytdl.py

```diff
@@ -113,14 +118,19 @@ async def update_status(self):
             if status is None:
                 return
             self.tmpfilename = status.get('tmpfilename')
+            if 'filename' in status:
+                self.info.filename = os.path.basename(status.get('filename'))

```

Comment:
sometimes the OUTPUT_TEMPLATE might be set in such a way that downloads are put into directories (see for example here: https://github.com/alexta69/metube/issues/45), in which case this code won't give the right result...

Reply:
Didn't think of that. Good point.

Reply:
Do you think we can handle it in some way so that the feature works in these cases too?

Reply:
Already pushed an update that uses relpath. Tested with OUTPUT_TEMPLATE from the issue you referenced.

Reply:
I see now. It's best to push as additional commits, because then it's easier to see what changed since last time. But doesn't matter right now.